codemp/rd-rend2/tr_extensions.cpp

```diff
@@ -256,6 +256,10 @@ static qboolean GetGLFunction ( GLFuncType& glFunction, const char *glFunctionSt
 	return qtrue;
 }
 
+#ifndef WIN32 // [M] there's no __stdcall on linux
```

Comment:
Don't need this. The `__stdcall` below can be changed to `QCALL` (https://github.com/xycaleth/OpenJK/blob/master/shared/qcommon/q_platform.h#L29)